cdc_rsync_server/main.cc

```diff
@@ -60,10 +70,19 @@ ServerExitCode GetExitCode(const absl::Status& status) {
 }  // namespace cdc_ft
 
 int main(int argc, const char** argv) {
-  if (argc < 2) {
-    printf("Usage: cdc_rsync_server <port> cdc_rsync_server <size> <time> \n");
-    printf("       where <size> and <time> are the file size and modified\n");
-    printf("       timestamp (Unix epoch) of the corresponding component.\n");
+  if (argc == 2 && strcmp(argv[1], "--version") == 0) {
+    printf("%s\n", BUILD_VERSION);
+    return 0;
+  }
+
+  if (argc < 5) {
+    printf(
+        "Usage: cdc_rsync_server <port> cdc_rsync_server <size> <time> "
```

Comment:
Move the <build_id> before cdc_rsync_server and adjust the line scanning for components:
cdc_ft::GameletComponent::FromCommandLineArgs(argc - 2, argv + 2);
to -3 and +3, resp. GameletComponent::FromCommandLineArgs assumes that all remaining args belong to components. It probably just works as it ignores args if there are less than 3 left, but it's more a coincidence.